dll/win32/imm32/imm.c

```diff
@@ -5096,8 +5096,18 @@ BOOL WINAPI DllMain(HINSTANCE hinstDLL, DWORD fdwReason, LPVOID lpReserved)
             break;
 
         case DLL_THREAD_DETACH:
+#if 1
+            /* FIXME: hack */
```

Comment:
Is there an `Unregister` alternative to `ImmRegisterClient` where g_psi should be set to zero?

Reply:
MS seems to call a user a "client". Probably this "registering client" means registering `user32` module.